lib/apollo-federation/tracing.rb

```diff
@@ -9,13 +9,24 @@ module Tracing
     module_function
 
     def use(schema)
-      schema.tracer ApolloFederation::Tracing::Tracer
+      if silence_deprecation_warning?
+        schema.tracer ApolloFederation::Tracing::Tracer, silence_deprecation_warning: true
```

Comment:
```suggestion
        schema.tracer ApolloFederation::Tracing::Tracer, silence_deprecation_warning: silence_deprecation_warning?
```
no strong opinion, but wondering why not ☝️

Reply:
@lucasvianna Grphql bellow version 2.3.0 does not accept the second argument